docs/tutorials/Canopy/soil_canopy_tutorial.jl

```diff
@@ -109,6 +110,67 @@ include(
 # Define the parameters for the soil model and provide them to the model 
 # parameters struct:
 
+# Heterotrophic respiration parameters
+P_sfc = FT(101e3)
```

Comment:
Im not sure this should be a parameter - should we get it from the PrescribedAtmos struct instead?

Reply:
I agree. 
I started working on this - this will modify the biogeochemistry model (deleting a parameter, adding a driver), so there will be a bunch of edits required throughout the repo